src/main/java/frc/robot/subsystems/SuperStructure.java

```diff
@@ -79,13 +80,24 @@ public Command stow() {
         spinnyClaw.stop());
   }
 
+  public Command preIntake() {
+    return Commands.sequence(
+        stow(),
```

Comment:
Why does it need to move to the stowed position first (arm angled up) and then move to the pre-intake position (arm pointing straight down)?

Reply:
It probably doesn't, my mistake from when I was telling him how to write the sequence. (I originally had `stow()` in the beginning so that it would act the same as the old intake sequence except the pre-intake happened without needing to press the button, but we can just directly go into the pre-intake position)